.github/workflows/e2e_android.yaml

```diff
@@ -45,7 +45,7 @@ jobs:
       - uses: actions/setup-java@cd89f46ac9d01407894225f350157564c9c7cee2 # 3.12.0
         with:
           distribution: "corretto" # Amazon Corretto Build of OpenJDK
-          java-version: "11"
+          java-version: "17"
```

Comment:
why to update the java version? I think the java version used in our android packages are still version 8 and so not sure why to use different version on CI.

Reply:
The newer gradle version to 8.1.0 requires java 17 and above to run.

Reply:
I see, so I think we should update the java version and jvmTraget in the android packages as well